config.yml

```diff
@@ -1,21 +1,21 @@
 # These options apply to how the script will operate.
 script_options:
-  # Switch between running in test mode (no buying/selling) and live mode
+  # Switch between testnet and mainnet
   # Setting this to False will use REAL funds, use at your own risk
-  TEST_MODE: True
```

Comment:
Should probably have TEST_MODE = True as default config

Reply:
Sure, seems logical based on the current userbase :-)